client/src/components/web-extension/Common/AuthLogin/index.jsx

```diff
@@ -94,18 +89,11 @@ const AuthLogin = ({
 							<Form.Text className="invalid-feedback" id="passwordHelpBlock">
 								{serverErrors.message}
 							</Form.Text>
-						)}  
+						)}
 						<div className="cd_we_page--bottom">
 							<Button type="submit" className="cd_we_btn-next" disabled={false}>
 								Unlock
 							</Button>
-							{isShowReset && (
```

Comment:
`<AuthLogin />` is a shared password entering component that is shared  across  2 places:
- Case 1 **Welcome Back** screen: On locking, User will be redirected back to /WelcomeBack screen. They have 2 choices:
- - Enter password to login again to their previous account
- - Choose to exit completely, "Log in as another User" by importing different keyphrase.
- Case 2 Session Expired or Service Worker interrupted: This will force User to enter password again. In this case there will be no "Login as another User" as `isShowReset` is set to `false`

Reply:
It's intentional to remove the `Log in as another User` for now, as we want user to enter password to delete data, similar with `delete all` button.

Reply:
Good one, now it's clear